masterlist.yaml

```
@@ -8042,6 +8042,20 @@ plugins:
       - crc: 0x734AD62B
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'Skyrimesque Counts and Countesses.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/43420/' ]
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Requires the meshes and textures from [Slof''s Victorian Clothes](https://www.nexusmods.com/oblivion/mods/49002/).'
```

This message can be communicated using an existing anchor, namely `&requiresResources`, & you can check for the folders installed by the mod since they're uniquely enough named. Also, since the message is now conditional, it can be upgraded to a warning (which the anchor is by default).

```suggestion
      - <<: *requiresResources
        subs: [ '[Slof''s Victorian Clothes](https://www.nexusmods.com/oblivion/mods/49002/)' ]
        condition: 'not file("meshes/asvictorian") or not file("textures/asvictorian") or not file("textures/menus/icons/asvictorian")'
```